docs/docs/configuration/auth.md

```diff
@@ -104,6 +105,34 @@ in the App registration manifest file.
    --oidc-issuer-url=https://login.microsoftonline.com/{tenant-id}/v2.0
 ```
 
+#### Authentication with federated credentials on Kubernetes
```

Comment:
I think this would be the prefered method? Should be above the long lived variant.